pkg/tasks/worker/worker.go

```diff
@@ -153,18 +154,22 @@ func (w *worker) process(ctx context.Context, taskInfo *models.TaskInfo) {
 	defer recoverOnPanic(*logger)
 
 	if handler, ok := w.handlers[taskInfo.Typename]; ok {
+		var finishStr string
 		err := handler(ctx, taskInfo, &w.queue)
 		if err != nil {
+			finishStr = fmt.Sprintf("task failed with error: %v", err)
+			logger.Warn().Msgf("error during task processing: %v", err)
```

Comment:
this actually results in the error getting logged twice for me:
```
4:47PM WRN error during task processing: traceback:   File "/usr/local/lib/python3.8/site-packages/pulpcore/tasking/tasks.py", line 65, in _execute_task

```

```
4:47PM INF [Finished Task] task failed with error: traceback:   File "/usr/local/lib/python3.8/site-packages/pulpcore/tasking/tasks.py", line 65, in _execute_task
```

Reply:
one is info and one is warn.  Could we just do it once (severity could change based on if its failed or not)

Reply:
fixed